[RFR] Add documentation for allowMissing Polyglot option (#3930)
marmelab/react-admin: docs/Translation.md

}
```

## Translating Notification Messages

> By default, react-admin translates the notificaiton messages. You can pass variables for polyglot interpolation with custom notifications. For example:

Reviewer: ```suggestion
By default, react-admin translates the notification messages. You can pass variables for polyglot interpolation with custom notifications. For example:
```

    },
};
```

## Silencing Translation Warnings

> By default, the `polyglotI18nProvider` logs a warning in the console each time it is called with a message that can't be found in the current translations. This is a Polyglot feature that helpts tracking missing translation messages.

Reviewer: ```suggestion
By default, the `polyglotI18nProvider` logs a warning in the console each time it is called with a message that can't be found in the current translations. This is a Polyglot feature that helps tracking missing translation messages.
```

> But you may want to avoid this for some messages, e.g. error messages from a data sourc eyou don't control (like a web server).

Reviewer: ```suggestion
But you may want to avoid this for some messages, e.g. error messages from a data source you don't control (like a web server).
```
